### ISSUE TITLE
Error in Leaderboards query

### ISSUE THREAD

**Issue by riztaha** [LHL-FocusPocus/FocusPocus#59]
Query is not returning accurate sum of all durations.

Need to change where query starts and possibly the route it joins the tables.
Need to look into nested queries.

(Find all users bad browsing history over the past week. Sum each individual's durations over the week. Group by each individual. Order by time.)

**Comment by riztaha** [comment]
Tried & researched different queries to achieve goal. goal = null.

Will try to seek help on this from James.

**Comment by jpqy** [comment]
Fixed it:
```
SELECT first_name as name, users.id, SUM(duration) as time
        FROM browse_times
        JOIN users on browse_times.user_id = users.id
        JOIN blacklists ON users.id = blacklists.user_id AND blacklists.website_id = browse_times.website_id
        WHERE datetime_start >= CURRENT_DATE - INTERVAL '7 days'
        AND datetime_start < CURRENT_DATE + INTERVAL '1 day'        
        GROUP BY users.id
        ORDER BY time
        DESC LIMIT 6;
```

You forgot to join the website_id from blacklists and browse_times as part of your joins.

**Comment by jpqy** [comment]
Also didn't need `AND browse_times.website_id != 0` since website_id of 0 won't show up on anyone's blacklists.

**Comment by jpqy** [comment]
Fixed as part of PR #58
